Brocfile.js

```diff
@@ -1,6 +1,8 @@
 var merge = require('broccoli-merge-trees');
 var compileSass = require('broccoli-sass');
 
+ var includepaths = [ 'assets', 'bower_components', 'bower_components/fontawesome' 'bower_components/neat/app/assets/stylesheets' 'bower_components/app/assets/stylesheets' ]
+
 var appCss = compileSass(['assets', 'bower_components'], 'app.scss', 'app.css');
```

Comment:
You have created an `includesPaths` variable and then you didn't use it? Instead you passed the literal array `['assets', 'bower_components']`. Try replacing that with `includePaths` (after fixing the issue above).